[2024-02-28 00:15:18,370] {/Users/raghavmaini/Desktop/rttMVP/rttDataService/rtt_data_app/utils/post.py:130} ERROR - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper[PostUpvote(PostUpvote)]'. Original exception was: Error creating backref 'votes' on relationship 'PostUpvote.post': property of that name exists on mapper 'Mapper[Post(Post)]'
[2024-02-28 00:15:18,372] {/Users/raghavmaini/Desktop/rttMVP/rttDataService/rtt_data_app/utils/post.py:130} ERROR - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'Mapper[PostUpvote(PostUpvote)]'. Original exception was: Error creating backref 'votes' on relationship 'PostUpvote.post': property of that name exists on mapper 'Mapper[Post(Post)]'
